crates/backend/src/error.rs

```diff
@@ -30,6 +30,7 @@ enum Repr {
         text: String,
         span: Option<(Span, Span)>,
     },
+    Darling(TokenStream),
```

Comment:
This is not the right way to do this, but `darling` will likely need a change of some sort to fix this.

* There isn't an `impl ToTokens for darling::Error`, so we can't keep the error as-is in the `Repr` enum
* There's no way to read the span of a `darling::Error`, so we can't construct a `Single` from it (and therefore can't flatten them into a `Multi`)

Reply:
The necessary change is now up as TedDriggs/darling#184.

Reply:
This has been updated, and no longer needs to change `backend::Error`'s internals.